message/message_matcher.go

```diff
@@ -98,6 +98,26 @@ func getNumericValue(msg *Message, stmt *Statement) float64 {
 	return 0
 }
 
+func getTimestampValue(msg *Message, stmt *Statement) float64 {
```

Comment:
The matcher shouldn't have to know anything about this.  The time should always be converted to a numeric value by the grammar and everything should work as it did before.